### PR TITLE
Added SPLENV_REF parameter to official-release job

### PR DESCRIPTION
**Pull request by gcomoretto**

### CODE REVIEW

File: pipelines/lib/util.groovy [hunk 1]
```diff
@@ -1603,17 +1609,24 @@ def Object runBuildStack(Map p) {
     DOCKER_TAGS: '',  // null is not a valid value for a string param
   ] + p.parameters
 
-  def result = build(
-    job: p.job,
-    parameters: [
+  def jobParameters = [
       string(name: 'PRODUCTS', value: p.parameters.PRODUCTS),
```
Comment (ktlim) [inline pipelines/lib/util.groovy:1613]: Reduce the indent.

File: jobs/official_release.groovy [hunk 2]
```diff
@@ -18,7 +21,10 @@ p.pipeline().with {
       'existing git ref(s) upon which to base the release. Eg. "w.9999.52"')
     stringParam('RELEASE_GIT_TAG', null,
       'git tag for the new release. Eg. "v888.0.0.rc1"')
+    stringParam('SPLENV_REF', scipipe.template.splenv_ref, 'conda env ref')
     booleanParam('O_LATEST', false,
       'update the eups "O_LATEST" tag -- should only be done for a final (non-rc) release')
+    booleanParam('NO_TRIGGERED_JOBS', false,
```
Comment (ktlim) [inline jobs/official_release.groovy:27]: Why would you not want to run the triggered jobs for an official release?  At a minimum, we should create the documentation and the docker containers, even if we don't run validate_drp and ap_verify.

Comment (gcomoretto) [reply jobs/official_release.groovy:27]: If I understand correctly the code, I think there is no way to pass the ENVREF to most of the triggered jobs.
I suggest keeping the change to the minimum required in order to be able to do the release and open a new issue to parameterized properly the triggered jobs.

**Comment by gcomoretto** [comment]
I have removed the extra parameter on the triggered job.

**Pull request merged**